pkgs/development/tools/kbld/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles, testers, kbld }:
+
+buildGoModule rec {
+  pname = "kbld";
+  version = "0.37.0";
+
+  src = fetchFromGitHub {
+    owner = "carvel-dev";
+    repo = "kbld";
+    rev = "v${version}";
+    sha256 = "sha256-AQ+4eF5wdL14I2JbZrcz6okEkHtBnyczU91aM5rGvpI=";
+  };
+
+  vendorHash = null;
+
+  subPackages = [ "cmd/kbld" ];
+
+  ldflags = [
+    "-X github.com/vmware-tanzu/carvel-kbld/pkg/kbld/version.Version=${version}"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  passthru.tests.version = testers.testVersion {
+    package = kbld;
+  };
+
+  meta = with lib; {
+    description = "kbld seamlessly incorporates image building and image pushing into your development and deployment workflows";
```

```suggestion
    description = "Seamlessly incorporate image building and image pushing into your development and deployment workflows";
```

From <https://nixos.org/manual/nixpkgs/stable/#var-meta-description>:

> 
> A short (one-line) description of the package. This is shown by `nix-env -q --description` and also on the Nixpkgs release pages.
> 
> Don't include a period at the end. Don't include newline characters. Capitalise the first character. For brevity, don't repeat the name of package --- just describe what it does.
> 
> Wrong: `"libpng is a library that allows you to decode PNG images."`
> 
> Right: `"A library for decoding PNG images"`